hac-node/agent/service.go

```
@@ -0,0 +1,372 @@
+		listenAddr: ListenAddr,
+	}
+	s.engine.POST("/getProposals", s.handleGetProposals)
+	s.engine.POST("/getDiscussions", s.handleGetDiscussions)
+	s.engine.POST("/getGrants", s.handleGetGrants)
+	s.engine.POST("/getAgents", s.handleGetAgents)
+	s.engine.POST("/getAgentDetail", s.handleGetAgentDetail)
+	return s
+}
+
+func (s *Service) Start() {
+	s.engine.Run(s.listenAddr)
```

golangci-lint (1.62.2)

Error return value of `s.engine.Run` is not checked

(errcheck)